src/styles/saas-list.less

```diff
@@ -1,53 +1,93 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    width: 100%;
+    padding: 20px;
+    padding-left: @grid-gutter-width;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
   .card-view-pf {
-    padding-left: 21px;
-  }
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    flex-direction: column;
+    padding: 0;
+    margin: 0 (@grid-gutter-width / 2);
 
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+    .card-content {
+      cursor: pointer;
+      margin: 0;
+      display: flex;
 
-    @media (min-width: @screen-sm-min) {
-      margin: 2px 0 10px;
+      & > div {
+        display: flex;
+      }
     }
-  }
 
-  .card-view-pf .card {
-    height: 100%;
-    width: 100%;
-    cursor: pointer;
-    text-align: inherit;
-    margin-bottom: 8px;
-
-    @media (min-width: @screen-sm-min) {
-      height: 200px;
-      width: 260px;
-      margin-bottom: 20px;
+    .card {
+      background: none;
+      box-shadow: none;
```

Comment:
It doesn't feel like we should need to set this. Is this coming from `card-view-pf`? Can we just leave that class off since they're no longer styled as cards?